depmapomics/ccle.py

```diff
@@ -0,0 +1,1423 @@
+        tokeep_wes (dict, optional): a dict of wes lines that are blacklisted on the tracker due to CN qc but we want to keep their mutation data. Defaults to RESCUE_FOR_MUTATION_WES.
+        tokeep_wgs (dict, optional): a dict of wgs lines that are blacklisted on the tracker due to CN qc but we want to keep their mutation data. Defaults to RESCUE_FOR_MUTATION_WGS.
+        prev (pd.df, optional): the previous release dataset (to do QC). 
+            Defaults to ccle =>(tc.get(name=TAIGA_ETERNAL, file='CCLE_mutations')).
+        minfreqtocall (float, optional): the minimum frequency to call a mutation. Defaults to 0.25.
+    """
+    from taigapy import TaigaClient
+
+    tc = TaigaClient()
+    if prev == "ccle":
+        prev = tc.get(name=TAIGA_ETERNAL, file="CCLE_mutations")
+    if doCleanup:
+        # TODO:
+        val = ""
+        # gcp.rmFiles('gs://fc-secure-012d088c-f039-4d36-bde5-ee9b1b76b912/$val/**/call-tumorMM_Task/*.cleaned.bam')
+    # sometimes it does not work so better check again
+
+    # doing wes
+    print("doing wes")
+    folder = os.path.join("output", samplesetname, "wes_")
+
+    wesmutations = mutations.postProcess(
+        wesrefworkspace,
+        AllSamplesetName if AllSamplesetName else samplesetname,
+        save_output=folder,
+        doCleanup=True,
+        mutCol="CGA_WES_AC",
+        **kwargs,
+    )
+
+    pr_table = trackerobj.read_pr_table()
+    renaming_dict = dict(list(zip(pr_table.CDSID, pr_table.index)))
+    wesmutations_pr = wesmutations[
+        wesmutations[SAMPLEID].isin(renaming_dict.keys())
+    ].replace({SAMPLEID: renaming_dict})
+    wesmutations_pr.to_csv(folder + "somatic_mutations_all_profile.csv", index=None)
+
+    # doing wgs
+    print("doing wgs")
+    folder = os.path.join("output", samplesetname, "wgs_")
+
+    wgsmutations = mutations.postProcess(
+        wgsrefworkspace,
+        sampleset="allcurrent",  # AllSamplesetName if AllSamplesetName else samplesetname,
+        save_output=folder,
+        doCleanup=True,
+        mutCol="CGA_WES_AC",
+        **kwargs,
+    )
+
+    wgsmutations_pr = wgsmutations[
+        wgsmutations[SAMPLEID].isin(renaming_dict.keys())
+    ].replace({SAMPLEID: renaming_dict})
+
+    wgsmutations_pr.to_csv(folder + "somatic_mutations_all_profile.csv", index=None)
+
+    # merge
+    print("merging")
+    folder = os.path.join("output", samplesetname, "merged_")
+    priomutations = wgsmutations_pr.append(wesmutations_pr).reset_index(drop=True)
+    # normals = set(ccle_refsamples[ccle_refsamples.primary_disease=="normal"].arxspan_id)
+    # mutations = mutations[~mutations[SAMPLEID].isin(normals)]
+    # priomutations.to_csv(folder + "somatic_mutations_profile.csv", index=False)
+
+    # making binary mutation matrices
+    print("creating mutation matrices")
+
+    merged = priomutations[priomutations["tumor_f"] > 0.05]
+    merged = mutations.annotateLikelyImmortalized(
```

Comment:
we might want that as part of the regular postProcessing

Reply:
That makes sense. Was there a reason why we put it here? If there's no other concerns I'll move it to the generic one

Reply:
remove line 958, and move calling annotateLikelyImmortalized to mutation generic func

Reply:
turns out we are already running this in the regular postprocessing function, so I'm removing the duplicate in the ccle one